ideascube/mediacenter/views.py

```diff
@@ -16,6 +16,9 @@
 from ideascube.decorators import staff_member_required
 from ideascube.mixins import FilterableViewMixin, OrderableViewMixin
 
+# Do not import directly Catalog as we want to mock it for test.
```

Comment:
You could still mock it even if you imported it directly, though. Why would it be a problem?